.changeset/eight-humans-sip.md

```diff
@@ -0,0 +1,7 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/i18n": patch
+"@rocket.chat/rest-typings": patch
```

Comment:
```suggestion
"@rocket.chat/meteor": minor
"@rocket.chat/i18n": minor
"@rocket.chat/rest-typings": minor
```

Are these really patches? Wondering because the PR is tagged as `feat`

Reply:
And this is what I got for using the changeset comment on here 🤣 

Good one